api/db/changelog/db.changelog-60-backfill-bypass.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd
+        http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd">
+  <changeSet author="blrubenstein" id="changelog-60-backfill-bypass">
+    <sql>UPDATE user
```

Comment:
There's an argument to backfill `data_use_agreement_bypass_time` here as well; I'll defer to @gjuggler on that.

Reply:
Fair. We will wait to hear from Greg

Reply:
If we don't actually have the data use agreement click-through set up (which I believe is the case?) then we shouldn't backfill here. Presumably asking users to actually click through the agreement once we do implement & enable that module won't be an excessive burden.